### ISSUE TITLE
 Is this a dead project?

### ISSUE THREAD

**Issue by weslien** [indeedeng/iwf-ts-sdk#12]
Looking at iwf as a possible abstraction over temporal. go sdk looks great, but since we have a lot of nodejs stuff to migrate. it would be awesome if this sdk were actually available. Looks to be in limbo atm though.

Gustav@Lolo

**Comment by longquanzheng** [comment]
No, we will continue on it. I need to learn node Js. Maybe I will take a look later when I get time（maybe Jan of 2025)

**Comment by vandervillain** [comment]
This SDK is now live and published to npm at https://www.npmjs.com/package/iwf-ts-sdk
